recipes/mauvealigner/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "mauvealigner" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  skip: True [osx]
+  number: 0
+
+source:
+  url: https://depot.galaxyproject.org/software/mauveAligner/mauveAligner_1.2.0_src_all.zip
+  sha256: 0d0cf5dd560cf062d4bfb01b121b38865f3a7fae6312d4169eb696add82cefc7
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - autoconf
+    - automake
+    - libtool
+    - libgenome
```

Comment:
libgenome and the others below needs to be in the host section, I think

Reply:
ooo...thanks...Is this a new high-speed linting service?